MD5/MD5.Tests/CheckSumTests.cs

```diff
@@ -0,0 +1,40 @@
+namespace MD5.Tests;
+
+public class Tests
+{
+    public Tests()
+    {
+        Directory.SetCurrentDirectory("../../../..");
+    }
+
+    [Test]
+    public void ParallelAndNotParallelShouldSameResult()
```

Comment:
```suggestion
    public void ParallelAndNotParallelShouldProduceSameResult()
```
Или ка-то так, иначе не очень по-английски